python/googleapis/python-multi/Dockerfile

```diff
@@ -129,4 +129,22 @@ ENV PATH /google-cloud-sdk/bin:$PATH
 # Enable redis-server on boot.
 RUN sudo systemctl enable redis-server.service
 
+# Install docker
+RUN apt-get update \
+  && apt-get install -y \
+     apt-transport-https \
+     ca-certificates \
+     curl \
+     software-properties-common \
+     sudo
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+
+RUN apt-get update && apt-get install -y docker-ce
```

Comment:
I think you can probably move this into the install chain above as well, you'll just need to move the key and add-apt/repository higher up in the dockerfile.

Reply:
To install docker I need `curl` and `sudo` packages, is it ok to move the dependencies to the chain as suggested in previous comment and have a separate section for Docker itself?

```
# Install docker
RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -

RUN add-apt-repository \
   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
   $(lsb_release -cs) \
   stable"

RUN apt-get update && apt-get install -y docker-ce
```

Reply:
That seems reasonable to me - you probably should clear the apt-get caches and stuff like the above command does though

Reply:
Thanks, just submitted update.